src/main/java/org/jenkinsci/plugins/xunit/threshold/FailedThreshold.java

```diff
@@ -78,4 +95,139 @@ public boolean isValidThreshold(double threshold, double value) {
         return value <= threshold;
     }
 
+    private int getQuarantined(XUnitLog log, TestResult testResultAction, FilePath workspace) {
+
+        if (workspace == null)
+            return 0;
+
+        String quarantinedTestsFileName = "quarantined-tests.json";
+        log.info(String.format("Searching workspace `%s` for %s files." , workspace, quarantinedTestsFileName));
+
+        List<TestSetting> listOfQuaratinedTests = new ArrayList<>();
+
+        try
+        {
+            if(!workspace.isRemote())
```

Comment:
this has to be tested on linux!

Reply:
Are we confident that all files are copied to jenkins workspace; i.e., no remote access?

Reply:
nope, needs testing.